gnomad/resources/resource_utils.py

```diff
@@ -109,16 +109,23 @@ class TableResource(BaseResource):
 
     expected_file_extensions: List[str] = [".ht"]
 
-    def ht(self, force_import: bool = False) -> hl.Table:
+    def ht(
+        self,
+        force_import: bool = False,
+        read_args: Optional[Dict[str, Any]] = None,
+    ) -> hl.Table:
         """
         Read and return the Hail Table resource.
 
+        :param force_import: If ``True``, force the import of the resource even if it
```

Comment:
as a basic question, why is this script the only place in the repo where True is wrapped in two backticks in docstrings? (is this the format we should be using for our other docstrings?)

Reply:
Yeah I'm not completely sure either. It was suggested by co-pilot and was already used in other parts of this file. It looks like maybe we should be using two, but I'm not positive.

https://documatt.com/restructuredtext-reference/element/inline-literal.html

```
Common issues:
Don’t interchange ` and ``
There is a big difference between `code example` and ``code example``! Only the latter with double `` is inline code example! Text wrapped in single ` is so-called default role interpreted text.

This is tricky part of reStructuredText because in Markdown, code examples are in wrapped in single backtick.
```

[Here is what the two backticks looks like](https://broadinstitute.github.io/gnomad_methods/api_reference/resources/resource_utils.html#gnomad.resources.resource_utils.BaseResource)

Reply:
whoa, TIL. I like how the double backticks looks in our docs, so definitely happy to adopt this practice moving forward